### PR TITLE
Issue-9  Added location swticher functionalities and Added a fix on fetching active location from Session

### PR DESCRIPTION
**Pull request by sumangala028**
I have fixed the location switcher issues. I have tested the implementation locally, and it worked for me without any issues.

 Please find the fixes below,

1. Active current location will be fetched from the "appui/session" end point.
2. Location changes will be sent to the "appui/session" to replace the current active location information.

Issue : #9

### CODE REVIEW

File: app/js/components/common/Header.jsx [hunk 1]
```diff
@@ -48,7 +57,15 @@ export class Header extends Component {
         this.setState({currentLocationTag: e.target.id});
     }
 
-    dropDownMenu(locationTags) {
+    setLocation(e, locationUUIDTag) {
+        this.setState({currentLocationTag: e.target.id});
+        const apiHelper = new ApiHelper(null);
+        apiHelper.post("/appui/session", {"location" : locationUUIDTag}).then(response => {
+            // nothing to do
```
Comment (JudeNiroshan) [inline app/js/components/common/Header.jsx]: It's better to leave a comment like 'Re-render the location based components in UI'

**Review by JudeNiroshan — CHANGES_REQUESTED** [review]
Please make the change of that comment. So that I can merge this. 
Make sure in your PRs, you need to verify that you have done the testing before making a PR. It should be stated in your PR. :-)

File: app/js/components/common/Header.jsx (hunk 1 above)
Comment (sumangala028) [reply app/js/components/common/Header.jsx]: Changed

**Comment by sumangala028** [comment]
Hi @JudeNiroshan, 
I have changed that comment. I have tested the implementation locally and It worked for me with out any issues.

**Comment by JudeNiroshan** [comment]
@sumangala028 could you please resolve the Codacy quality review issues in this PR?

**Comment by sumangala028** [comment]
Sure, I will resume the work on this PR.

**Comment by sumangala028** [comment]
@JudeNiroshan  I have fixed some Codacy issues, and others couldn't be replaced with any methods. Could you please review it?

**Comment by JudeNiroshan** [comment]
@sumangala028 , Ridmal has done some changes for eslint compatibility. I think, it would be better you take those updates and do changes on top of it. Also it would be easier if you create a new PR which will be easy to keep track. :-)

**Comment by sumangala028** [comment]
Sure, I will update my PR :-)

**Comment by sumangala028** [comment]
I will create a new PR to get updates from the eslint.

**Pull request closed without merging**